transform-environment

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+"""
+Transform environment for Document Viewer uploads.
+
+Usage:
+    transform-environment --input=<input> --output=<output> --environment=<environment>
+
+  
+Options:
+    -h --help           Show this helpful output
+    --input=<input> file to read from
+    --output=<output> file to write to
+    --environment=<environment> environment to write
+"""
+
+from docopt import docopt
+
+def main(args):
+    input_file = open(args["--input"], 'r')
+    input_data = input_file.read()
+    input_file.close()
+
+    preprod_input_string = f'\"preprod\\'
```

Comment:
I tightened this replacement a bit--it turns out that `\"preprod\\` only appears in the three locations we want to explicitly replace. All 46 other instances of the term `preprod` appear to be harmless as part of `src` attributes for the T logo, but this will help minimize unwanted side effects.